include/argparse.hpp

```diff
@@ -800,6 +800,14 @@ class ArgumentParser {
     }
   }
 
+  void add_pre_text(std::string aPreText) {
+    mPreText = std::move(aPreText);
+  }
+
+  void add_post_text(std::string aPostText) {
+    mPostText = std::move(aPostText);
+  }
+
```

Comment:
Similar to Python, let's change these method names (and members) to [description](https://docs.python.org/2/library/argparse.html#description) and [epilog](https://docs.python.org/2/library/argparse.html#)

`parser.add_description(value)` and `parser.add_epilog(value)` 

* `description`:  This argument gives a brief description of what the program does and how it works. In help messages, the description is displayed between the command-line usage string and the help messages for the various arguments.
* `epilog`: Some programs like to display additional description of the program after the description of the arguments. Such text can be specified using epilog.